### ISSUE TITLE
Autobind action operations

### ISSUE THREAD

**Issue by nhunzaker** [vigetlabs/microcosm#144]
It would be nice to auto-bind `action.close`, `action.reject`, etc..., so that they can be passed around easier, like:

``` javascript
import promiseAjax from 'somewhere'

function searchAction (query) {
  return function (action) {
    action.open()
    promiseAjax.get('/search', query).then(action.close, action.reject)
  }
}
```

**Comment by nhunzaker** [comment]
Going to close this. I can't figure out a way to do it without hindering perf.

**Comment by nhunzaker** [comment]
Opening this up again. Let's analyze the perf and figure out how to make it work. This would be a big ergonomics win.

cc @greypants

**Comment by nhunzaker** [comment]
Landed in `microcosm@12.7.0-alpha.4`
